scripts/build/fiwarelab/build-fiwarelab-image.sh

```diff
@@ -0,0 +1,136 @@
+#/bin/bash
+
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# We assume the following preconditions in the place where this script runs:
+#
+# - The script runs *inside* base image base_centos_6 type
+# - The user that runs the script has sudoers permissions
+# - The following commands are installed and accesible in the default PATH: yum, 
+#    mktemp, chkconfig, iptables
+
+# Install welcome message
+MOTD=$(mktemp)
+echo ' ' > $MOTD
```

Comment:
You could have saved you a lot of effort with [heredocs](http://tldp.org/LDP/abs/html/here-docs.html).